pyupgrade.py

```diff
@@ -1785,6 +1785,54 @@ def _fix_fstrings(contents_text):
     return tokens_to_src(tokens)
 
 
+class FindOSErrorAliases(ast.NodeVisitor):
```

Comment:
let's combine this with the Py3Plus visitor -- each visitor we add adds ~30% execution time, but if we batch them up it's ~1-3% instead

Reply:
Sure

Reply:
What's the slow part btw? Finding the nodes or parsing from src to tokens and back? I wonder if there's a way to parallelize the work in a way that you could write a single "finder/replacer" each thing to rewrite, instead of having one big one, so you could keep the implementation fast but still manageable

Reply:
most of it comes from ast parse, the tokenize-untokenize is not the fastest as well